pkg/k8shandler/elasticsearch_helper.go

```diff
@@ -101,20 +101,24 @@ func updateAllIndexReplicas(clusterName, namespace string, client client.Client,
 
 	// get list of indices and call updateIndexReplicas for each one
 	for index, health := range indexHealth {
-		// only update replicas for indices that don't have same replica count
-		if numberOfReplicas := parseString("settings.index.number_of_replicas", health.(map[string]interface{})); numberOfReplicas != "" {
-			currentReplicas, err := strconv.ParseInt(numberOfReplicas, 10, 32)
-			if err != nil {
-				return false, err
-			}
+		if _, ok := health.(map[string]interface{}); ok {
```

Comment:
what does the response look like coming back in this case?

Reply:
this is response to `*/_settings/index.number_of_replicas` so it should be a map of map perhaps.

but if elasticsearch is not started yet, we may get a string response "opendistro not initialized yet" OR "admin user not allwed to login via http"

hence the type check.

Reply:
the readiness probe will block that though... i think this is the auth issue we're seeing